build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -252,54 +252,43 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
             return c.Success();
         }
 
-        [Target(nameof(CompileTargets.GenerateStubHostPackages))]
-        public static BuildTargetResult PackagePkgProjects(BuildTargetContext c)
+        [Target]
+        public static BuildTargetResult GenerateMSbuildPropsFile(BuildTargetContext c)
         {
             var arch = IsWinx86 ? "x86" : "x64";
-            
+            var hostVersion = c.BuildContext.Get<HostVersion>("HostVersion");
+
+            var msbuildProps = new StringBuilder();
+
+            msbuildProps.AppendLine(@"<Project xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"">");
+            msbuildProps.AppendLine("  <PropertyGroup>");
+            msbuildProps.AppendLine($"    <Platform>{arch}</Platform>");
+            msbuildProps.AppendLine($"    <DotNetHostBinDir>{Dirs.CorehostLatest}</DotNetHostBinDir>");
+            msbuildProps.AppendLine($"    <HostVersion>{hostVersion.LatestHostPolicyVersion.WithoutSuffix}</HostVersion>");
+            msbuildProps.AppendLine($"    <HostResolverVersion>{hostVersion.LatestHostFxrVersion.WithoutSuffix}</HostResolverVersion>");
+            msbuildProps.AppendLine($"    <HostPolicyVersion>{hostVersion.LatestHostVersion.WithoutSuffix}</HostPolicyVersion>");
+            msbuildProps.AppendLine($"    <BuildNumberMajor>{hostVersion.LatestHostBuildMajor}</BuildNumberMajor>");
+            msbuildProps.AppendLine($"    <BuildNumberMinor>{hostVersion.LatestHostBuildMinor}</BuildNumberMinor>");
```

Comment:
@weshaggard Can you help me understand how this translates to the version of Microsoft.NETCore.App?

We have an inconsistency here between the version of Microsoft.NETCore.App generated after this change and the version persisted in the shared framework redist causing portable apps to break.

There seems to be an additional `-00` appended. Is that expected and we should account for it in other places, or should we remove it here?